modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboBoxTest.java

```diff
@@ -653,9 +653,8 @@ public void defaultConverterCanHandleIncorrectType_2() {
     }
 
     @Test public void ensure_getPseudoClassStateReturnsValidValue() {
-        Stage stage = new Stage();
-        Scene scene = new Scene(comboBox);
-        stage.setScene(scene);
+        StageLoader sl = new StageLoader(comboBox);
+
 
```

Comment:
good catch :) But wouldn't show the stage have the same effect? (It's just me having a personal dislike of stageloader :)

Reply:
stage.show() has the same effect of fixing the bug. In fact, I fixed it initially with exact this fix.

When I looked at the other tests in ComboBoxTest class, I saw that there is a pattern of creating a Stage, creating a Scene with ComboBox and then adding Scene to the Stage. What was missed was stage.show(). The StageLoader does exactly these steps (including the missed stage.show()).
StageLoader might be a misfit at some of the other places in our tests (I believe that is the reason for your dislike), but I feel using StageLoader is a better fit in ComboBoxTest.

Reply:
thanks! 

Wondering as to when/why Stageloader might be a misfit? I completely gave up on using it a while ago, so don't really remember *cough

Reply:
It is not that StageLoader is bad as such. By misfit - I meant - error prone to be used in tests that already use their own Scene and Stage creation.

Reply:
ahh .. I see, thanks :)